contrib/examples/apiserver/serviceclass.yaml

```diff
@@ -3,7 +3,7 @@ kind: ServiceClass
 metadata:
   name: d35b55b2-b1fd-4123-8045-5b9c619cb629
 spec:
-  brokerName: test-broker
+  serviceBrokerName: test-broker
```

Comment:
this is going to be clusterServiceBrokerName? Since I'm doing the rename right now for
serviceBroker -> clusterServiceBroker, maybe hold off on this and I'll tack onto that rename?

Reply:
And / or , instead of a name, if we're going to have namespaced brokers, name is not going to cut it and this needs to be a ref more than likely and not a name?

Reply:
I don't want to transform any types right now, for the record.

Also, references between cluster- and namespaced-scoped brokers/plans/classes should not be a thing.